libs/base/Language/Reflection/TTImp.idr

```diff
@@ -435,7 +435,7 @@ Eq TTImp where
     c == c' && (assert_total $ i == i') && n == n' && a == a' && r == r'
   ILet _ _ c n ty val s == ILet _ _ c' n' ty' val' s' =
     c == c' && n == n' && ty == ty' && val == val' && s == s'
-  ICase _ t ty cs == ICase _ t' ty' cs'
+  ICase _ _ t ty cs == ICase _ _ t' ty' cs'
     = t == t' && ty == ty' && (assert_total $ cs == cs')
```

Comment:
I would have expected quantity `c` to part of the equality.

Reply:
Yeah, thanks for catching that.